app/models/household.rb

```diff
@@ -82,10 +88,16 @@ def build_household_coverage_member(family_member)
     primary_person = family_member.family.primary_person
 
     if Family::IMMEDIATE_FAMILY.include?(primary_relationship)
+      Rails.logger.info("BUILD: Adding ID: #{family_member.id} to immediate family for family_hbx_assigned_id: #{family.hbx_assigned_id}")
       immediate_family_coverage_household.add_coverage_household_member(family_member)
+
+      Rails.logger.info("BUILD: Removing ID: #{family_member.id} from extended family for family_hbx_assigned_id: #{family.hbx_assigned_id}")
       extended_family_coverage_household.remove_family_member(family_member)
     else
+      Rails.logger.info("BUILD: Removing ID: #{family_member.id} from immediate family for family_hbx_assigned_id: #{family.hbx_assigned_id}")
```

Comment:
@jayreddy519 do we need this logger? I'm just concerned about logging too much in this method.